config/jobs/image-pushing/k8s-staging-e2e-test-images.sh

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env bash
```

Comment:
iffy on this file location but I'll allow it I guess :shrug:

Reply:
copied from the original bash file

Reply:
I'm not happy with the sludge either but I don't have a better solution off the top of my head.  We could use a pair of verify/update `hack/` scripts to generate jobs using scripts stored in a well-known location?